### PR TITLE
fix: Added beta tag to decoupled loading screen

### PR DESCRIPTION
**Pull request by dalkia**
## What does this PR change?

Adds a beta tag to the decoupled loading screen

Also, added the feature flag to stop the loading message to update when the decoupled loading screen flag is on

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

File: unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/LoadingFeedbackController.cs [hunk 1]
```diff
@@ -35,20 +37,33 @@ private class SceneLoadingStatus
...
-            Environment.i.world.sceneController.OnNewSceneAdded += SceneController_OnNewSceneAdded;
-            GLTFComponent.OnDownloadingProgressUpdate += GLTFComponent_OnDownloadingProgressUpdate;
-            AssetPromise_AB.OnDownloadingProgressUpdate += AssetPromise_AB_OnDownloadingProgressUpdate;
-            CommonScriptableObjects.rendererState.OnChange += RendererState_OnChange;
+                Environment.i.world.sceneController.OnNewSceneAdded += SceneController_OnNewSceneAdded;
+                GLTFComponent.OnDownloadingProgressUpdate += GLTFComponent_OnDownloadingProgressUpdate;
+                AssetPromise_AB.OnDownloadingProgressUpdate += AssetPromise_AB_OnDownloadingProgressUpdate;
+                CommonScriptableObjects.rendererState.OnChange += RendererState_OnChange;
+            }
+            DataStore.i.featureFlags.flags.OnChange -= FeatureFlagsSet;
```
Comment (popuz) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/LoadingFeedbackController.cs:55]: I would recommend to put this unsubscribtion at the start of this method

File: unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/LoadingFeedbackController.cs [hunk 2]
```diff
@@ -35,20 +37,33 @@ private class SceneLoadingStatus
 
         public LoadingFeedbackController()
         {
-            loadedScenes = new List<SceneLoadingStatus>();
+            DataStore.i.featureFlags.flags.OnChange += FeatureFlagsSet;
+        }
+
+        private void FeatureFlagsSet(FeatureFlag current, FeatureFlag previous)
```
Comment (popuz) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/LoadingFeedbackController.cs:43]: please rename `previous` to `_`

**Review by popuz — APPROVED** [review]
Looks good 🟢 Left some cosmetical suggestions. One questions - is it testable or it is hard to test?

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenView.cs [hunk 3]
```diff
@@ -26,10 +27,15 @@ public override void Start()
             base.Start();
             showHideAnimator.OnWillFinishStart += FadeInFinish;
 
+#if !UNITY_EDITOR
```
[the remark this answers is not in the document]
Comment (dalkia) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenView.cs:30]: Didnt know the existance of `Application.isEditor`, thanks! Changing it!

File: unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/LoadingFeedbackController.cs (hunk 1 above)
Comment (dalkia) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/LoadingFeedbackController.cs:55]: Done

File: unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/LoadingFeedbackController.cs (hunk 2 above)
Comment (dalkia) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/LoadingFeedbackController.cs:43]: Done

**Pull request merged**